packages/cli/src/databases/entities/ExecutionMetadata.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn, RelationId } from 'typeorm';
+import { ExecutionEntity } from './ExecutionEntity';
+
+@Entity()
+export class ExecutionMetadata {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@ManyToOne('ExecutionEntity', 'metadata', {
+		onDelete: 'CASCADE',
+	})
+	execution: ExecutionEntity;
+
+	@RelationId((executionMetadata: ExecutionMetadata) => executionMetadata.execution)
+	executionId: number;
+
+	@Column('text')
+	key: string;
```

Comment:
we should either use stricter typing on this column, and add an index for faster searching, or get move this data to a simple json column on the executions table. 
in this current form, the additional table/entity feels unnecessary.

Reply:
let me rephrase: Does this really need to be a separate table?

Reply:
Yes, I think so - you can save multiple key-value pairs for each execution. The goal is to allow users to save and read data during execution and afterwards, search for this data in the executions panel.

When querying, you can query by some of the keys (initial implementation is a single key-value search but this can be changed in the future)

![Screenshot 2023-03-16 at 13 28 52](https://user-images.githubusercontent.com/219272/225617305-e90f83c4-f7b9-4cc8-ab73-f4496585920d.png)